### DISCUSSION TITLE
fr/usage/cfg/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#8333] [Docs]
# fr/usage/cfg/

Maîtrise les paramètres YOLOv8 et les hyperparamètres pour améliorer les performances du modèle. Apprends à utiliser les commandes YOLO CLI  , à ajuster les paramètres d'entraînement et à optimiser les tâches et les modes YOLO .

https://docs.ultralytics.com/fr/usage/cfg/

**Comment by DimitriDiantos** [comment]
thanks for this page. It was helpful for me.

**Reply by pderrenger** [reply]
Hey there! 😊

I'm glad to hear the documentation was helpful for you! If you ever need to dive deeper or have any questions about configuring your YOLOv8 model, don't hesitate to reach out. Here's a quick example of how you can adjust the learning rate when training a model:

```bash
yolo train data=coco128.yaml model=yolov8n.pt epochs=100 lr0=0.01
```

Happy coding, and best of luck with your projects! If you need further assistance, we're just an issue away. 🚀

**Comment by DimitriDiantos** [comment]
Hi,
Thanks for your reply. Actually I'm still working on my master thesis and
the topic is Conversion, Optimization and Implementation of Yolo Models for
the Nvidia Jetson Nano in Airborne Applications.
It is difficult for me to transfer my trained model(best.pt) into the
Jetson Nano using tensorRT. Can you help me? That will be great.

Best regards
Dimitri

Le mar. 20 févr. 2024 à 20:04, Paula Derrenger ***@***.***> a
écrit :

**Reply by pderrenger** [reply]
@DimitriDiantos hi Dimitri,

It's great to hear about your master thesis topic! For deploying your `best.pt` model on the Nvidia Jetson Nano using TensorRT, you can follow these general steps:

1. Export your PyTorch model to ONNX format with the command:
   ```bash
   yolo export model=best.pt format=onnx
   ```
2. Use the `onnx2trt` tool provided by TensorRT to convert the ONNX model to TensorRT:
   ```bash
   onnx2trt best.onnx -o best.trt
   ```
3. Load the `.trt` model on your Jetson Nano for inference.

Make sure you have the required TensorRT libraries installed on your Jetson Nano. If you encounter any specific issues, feel free to share the error messages or roadblocks, and we'll do our best to assist you further.

Best of luck with your thesis! 🎓
